src/utilities/templates/flatpages/flatpage-delete.html

```diff
@@ -0,0 +1,15 @@
+{% extends "flatpages/base.html" %}
+{% load crispy_forms_tags %}
+
+{% block head_title%} Confirm Deletion | {% endblock%}
+
+{% block content %}
+    <form action="" method="post">
+        {% csrf_token %}
+
+        <p>Are you sure you want to delete this custom page?<p>
```

Comment:
Add some info about the page here, at minimum it's name so they know what they are confirming. Something like: "Are you sure you want to delete the custom flatpage titled: <a href="{{flatpage url here}}> {{flatpage title here}} </a>"

Reply:
Done
![image](https://user-images.githubusercontent.com/39788517/172518031-c0e6853e-b91e-48ad-8a59-fb953b0fdbee.png)